GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -271,8 +321,22 @@ gdjs.WebsocketDebuggerClient.prototype.sendProfilerOutput = function(
   );
 };
 
-// This is an alternative to JSON.stringify that ensure that circular reference
-// are replaced by a placeholder.
+/**
+ * A Function used to replace Cycles with a default name.
+ * @callback cycleReplacer
+ * @param {string | number} key - The key corresponding to the value.
+ * @param {any} value - The value.
+ */
+
+/** 
+ * This is an alternative to JSON.stringify that ensure that circular reference
+ * are replaced by a placeholder.
+ * @param {any} obj - The object to serialize.
+ * @param {Function} [replacer] - The value to replace unsafe values with?
```

Comment:
Why is there a "?"

Reply:
Because I was not sure and wanted to get your attention so you could confirm that description/correct it.

Reply:
Let's go for: `A function called for each property on the object or array being stringified, with the property key and its value, and that returns the new value. If not specified, values are not altered.`